src/Microsoft.Health.Fhir.Api/Features/Filters/ValidateExportRequestFilterAttribute.cs

```diff
@@ -111,7 +120,10 @@ private bool IsValidBasicExportRequestParam(string paramName)
 
         private bool IsValidAnonymizedExportRequestParam(HttpRequest request, string paramName)
         {
-            if (request.Path.StartsWithSegments(DefaultExportRequestPath, StringComparison.InvariantCultureIgnoreCase))
+            if (AnonymizedExportRequestPathRegexs.Any(pathRegex => Regex.IsMatch(
```

Comment:
>AnonymizedExportRequestPathRegexs [](start = 16, length = 33)

Can we not use regex here?
if you pass `ActionExecutingContext context` here, you can inspect context.ActionArguments.Parameters for a presence of expected parameter for action.

Reply:
Thanks for reminding, now not need this regex to do the anonymized export params check anymore.